### ISSUE TITLE
Need supporting copy image in clipboard

### ISSUE THREAD

**Issue by csfreebird** [atlas-engineer/next#632]
I can upload an image file, but cannot copy image from clipboard in memory.

**Comment by jmercouris** [comment]
Thank you for the report! We'll work on this as soon as we can.

**Comment by Ambrevar** [comment]
I didn't know we could paste images in a browser.
Do you have any example website?

**Comment by csfreebird** [comment]
slack support this feature.

**Comment by azzamsa** [comment]
Yes, I always use `flameshot`/`maim` send to my clipboard then paste immediately to github issue (without uploading the file).

I can't do this in `next`.

**Comment by jmercouris** [comment]
This is true :-)

no updates on this yet!

**Comment by aartaka** [comment]
WebKit have recently added the "Copy Image" context menu option for images. I'm not sure we can do anything beyond that here (well, having a dedicated command could be cool, but that still is mostly on the side of WebKit). Closing this as completed.
